packages/clay-layout/stories/index.tsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@clayui/css/lib/css/atlas.css';
+import {number} from '@storybook/addon-knobs';
+import {storiesOf} from '@storybook/react';
+import React from 'react';
+
+import ClayLayout from '../src';
+
+const exampleStyles = {
+	backgroundColor: 'rgba(86,61,124,.15)',
```

Comment:
Not sure if it's too much, maybe you can provide some insight @pat270, but what do you think about using some brand colors here? Adding some branding to storybook might make it look more "ours" if you know what I mean.

Reply:
This sounds good to me. Maybe #E7E7ED or #cdced9?